<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_enviar">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>27</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del Producto</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_registro_ica">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>122</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumBlockCount">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_valor_producto">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>189</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumBlockCount">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>163</y>
     <width>139</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Valor del producto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>90</y>
     <width>94</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Registro ICA</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_nombre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>517</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumBlockCount">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_frecuencia_aplicacion">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>122</y>
     <width>255</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumBlockCount">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Frecuencia de Aplicaci√≥n</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>271</x>
     <y>163</y>
     <width>68</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_cantidad">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>256</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumBlockCount">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_enviar_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QMessageBox" name="messageBox">
   <property name="windowTitle">
    <string>Alerta</string>
   </property>
   <property name="text">
    <string>Por favor, complete todos los campos.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
